ios/test_app.rb

```diff
@@ -160,12 +160,27 @@ def make_project!(xcodeproj, project_root, target_platform, options)
   FileUtils.mkdir_p(destination)
   FileUtils.cp_r(xcodeproj_src, destination)
   name, display_name, version, single_app = app_config(project_root)
+  xcschemes_path = File.join(xcodeproj_dst, 'xcshareddata', 'xcschemes')
+
+  # Read "metalAPIValidation from the app config and apply it to the xcscheme"
+  metal_api_validation = platform_config('metalAPIValidation', project_root, target_platform)
+  puts "metalAPIValidation: #{metal_api_validation}"
+  
+  if metal_api_validation
+    xcscheme = File.join(xcschemes_path, "ReactTestApp.xcscheme")
+    puts "xcscheme: #{xcscheme}"
+    xcscheme_content = File.read(xcscheme)
+    new_content = xcscheme_content.gsub(/^\s*enableGPUValidationMode\s*=\s*"1"\s*$/, '')
```

Comment:
Should we use REXML to parse the scheme instead of using regex?

Reply:
It would also be nice if you could paste the note about `enableGpuValidationMode` being inverted here.

Reply:
Sure. Is this spot in the code fine for where to do it? I just looked for where we touch the xcscheme.

Reply:
I think so. Let's extract the relevant bits to a `function configure_xcscheme` in `xcode.rb` though. That should make it more testable as well.